src/substrate/filters/enricher.ts

```diff
@@ -33,6 +34,30 @@ export async function Enrich(
       /**
        * Staking Events
        */
+      case EventKind.NewSession: { // @mir-nawaz please review this
+        const [ sessionIndex ] = event.data as unknown as [ SessionIndex ] & Codec
+        const validators = await api.query.session.validators<Vec<ValidatorId>>();
+        const currentEra = await api.query.staking.currentEra();
+        let exposure : Vec<Exposure>
+        // erasStakers(EraIndex, AccountId): Exposure -> api.query.staking.erasStakers
+        if (validators && currentEra.isSome) { // if currentEra isn't empty
+          validators.forEach(async (validator) => {
+            const tmp_exposure = await api.query.staking.erasStakers(currentEra, validator) as unknown as Exposure & Codec;
+            exposure.push(tmp_exposure)
+          })
+        }
+        return {
+          data: {
```

Comment:
I think it is also be good to add the currentEra to the event_data.

Reply:
I was thinking the same!